src/frontend/src/index.css

```diff
@@ -57,7 +57,6 @@
         display: block;
         list-style-type: disc;
         margin: 1em 0 1em;
-        padding-left: 40px;
```

Comment:
Do not remove this, instead u can override the padding where u are using the element

Reply:
changed sir